src/ui/control/navigation_control.js

```diff
@@ -129,6 +133,7 @@ class NavigationControl {
     _createButton(className: string, fn: () => mixed) {
         const a = DOM.create('button', className, this._container);
         a.type = 'button';
+        a.setAttribute('role', 'button');
```

Comment:
This seems redundant as `<button>` elements have an [implicit `role` of `button`](https://www.w3.org/TR/html-aria/#el-button).